### PR TITLE
BaSP-MR-121: Error onBlur in forms inputs for Trainers

### PR DESCRIPTION
**Pull request by Emanuel-Lamberti**
It was possible to solve the error that error messages appeared under the inputs in the Trainers forms, both to add a new one and to edit an existing one.

### CODE REVIEW

File: src/Components/Trainers/TableTrainers/index.jsx [hunk 1]
```diff
@@ -4,22 +4,31 @@ import { ModalConfirm } from '../../Shared';
...
+  setShowFormEdit,
+  showFormEdit
+}) => {
   const [modalDeleteConfirmOpen, setModalDeleteConfirmOpen] = useState(false);
   const [modalSuccessOpen, setModalSuccessOpen] = useState(false);
   const [successMessage, setSuccessMessage] = useState(false);
   const [idTrainer, setIdTrainer] = useState('');
   const [trainerModify, setTrainerModify] = useState(null);
-  const [showForm, setShowForm] = useState(false);
+  //const [showForm, setShowForm] = useState(false);
```
Comment (maurojjzz) [inline src/Components/Trainers/TableTrainers/index.jsx:21]: remove commented lines

**Review by maurojjzz — CHANGES_REQUESTED** [review]

File: src/Components/Trainers/TableTrainers/index.jsx (hunk 1 above)
Comment (maurojjzz) [reply src/Components/Trainers/TableTrainers/index.jsx:21]: the rest seems ok

Comment (Emanuel-Lamberti) [reply src/Components/Trainers/TableTrainers/index.jsx:21]: I'll correct you, I'll pass it up.

**Comment by Emanuel-Lamberti** [comment]
I fixed the problem. I removed the comment line in the code.

**Review by maurojjzz — APPROVED** [review]
Nice work, seems fine.

**Review by FrancoLelli — APPROVED** [review]

**Pull request merged**